site/src/modules/resources/AgentRow.tsx

```diff
@@ -312,7 +295,8 @@ export const AgentRow: FC<AgentRowProps> = ({
 									width={width}
 									css={styles.startupLogs}
 									onScroll={handleLogScroll}
-									logs={startupLogs.map((l) => ({
+									overflowed={true}
```

Comment:
shouldn't this be 
```suggestion
									overflowed={agent.logs_overflowed}
```
?